labkey-client-api/src/org/labkey/remoteapi/storage/StorageRow.java

```diff
@@ -0,0 +1,71 @@
+package org.labkey.remoteapi.storage;
+
+import org.json.simple.JSONObject;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class StorageRow
+{
+    private String _type;
```

Comment:
Is it worth making an enum for this so users don't need to rely on the docs to find out valid values?

Reply:
done